js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
+const pushButton = document.querySelector(".first-task button");
+const outArea = document.querySelector(".first-task .out-area");
+const symbol = " $ "; // output symbol
```

Comment:
Please, rename it to 'outputSymbol' and remove comment.

Reply:
done